packages/bbui/src/Table/Table.svelte

```diff
@@ -373,6 +369,16 @@
                 class="spectrum-Table-cell"
                 class:spectrum-Table-cell--divider={!!schema[field].divider}
                 style={cellStyles[field]}
+                on:click={() => {
+                  if (!field.startsWith("custom-")) {
```

Comment:
Could we add a schema property to control this rather than depending on field name (which has the potential to cause false positives as unlikey as that is)? 

I was thinking we could add a `unselectable` property to the field schema, and if that new property is true then we skip the row selection here.

All we would need to do to implement that is in the client table component, add in `unselectable: true` when we construct the schema for the custom column, around line 85, and then check here
```js
if (!schema[field]?.unselectable) { ... }
```

There's also no reason for this to be 2 separate on:click handlers any more -  we can just execute both functions in the same handler now to save duplication,

Reply:
Yea that name is fine.

Not sure about the refactor around the fields though... what motivated you to do that? There's a bit of a strange mixture of `field` being the full field schema now, but yet there are many usages of `schema[field.name]` which are then redundant because that's just the same value as `field`. I would just have kept it as it was unless there's a reason to refactor it? Generally it's best to avoid changes to really core components like the table unless it's necessary.

Reply:
The `preventSelectRow` was in the schema though - so you could have accessed it like I mentioned in my original suggestion by just `schema[field]?.preventSelectRow`. We already access the field schema to pass into the `CellRenderer` just below that by `schema={schema[field]}`.

Reply:
No you're absolutely right - you even provided the code snippet in the first place which I managed to miss, apologies!
I'll get that sorted.